README.md

```diff
@@ -56,10 +61,10 @@ docker build -t linuxbrew-trusty https://raw.githubusercontent.com/Linuxbrew/doc
 
 ## Ubuntu 16.04 LTS (Xenial Xerus)
 ```sh
-docker run -it linuxbrew/linuxbrew
+docker build -t linuxbrew-xenial https://raw.githubusercontent.com/Linuxbrew/docker/master/xenial/Dockerfile
 ```
 
 ## Ubuntu 18.04 LTS (Bionic Beaver)
 ```sh
-docker run -it linuxbrew/linuxbrew
+docker run -it linuxbrew/brew
```

Comment:
_Technically_ this is now Homebrew/brew, but I'm not sure how maintained this repo is in general so maybe this particular nitpick is out of scope.

Reply:
`homebrew/brew` is preferred over `linuxbrew/brew`.

Reply:
This line should be
```dockerfile
docker build -t linuxbrew-bionic https://raw.githubusercontent.com/Linuxbrew/docker/master/bionic/Dockerfile
```

Reply:
`homebrew/brew` is based on Ubuntu 16.04 LTS (Xenial Xerus).